conf/systemd.service

```diff
@@ -5,7 +5,9 @@ After=network.target
 
 [Service]
 Type=simple
-User=root
+User=__APP__
+Group=__APP__
+ExecStartPre=__NODEJS__
```

Comment:
no need for _APP_ : https://github.com/YunoHost-Apps/cryptpad_ynh/blob/correction-patch3/scripts/_common.sh#L85